tdrs-backend/docker-compose.ci.yml

```diff
@@ -0,0 +1,18 @@
+# Docker compose file intended to be used in CI/CD pipelines
+version: "3"
+
+services:
+  web:
+    environment:
+      # Additional values passed in by the CircleCI host machine
+      - ACR_VALUES=${ACR_VALUES}
```

Comment:
I stripped out the `POSTGRES_x` variables that were being passed in here from CircleCI in the last PR so that they are instead pulled in from the base docker-compose.